FORMAT_ADIOS.md

```diff
@@ -32,3 +32,28 @@ Output from `bpls -A` for a boolean attribute `pybool` stored in the location of
 
 There is no convention yet for a unique representation of ADIOS2 variables with boolean type.
 Thus, implementations should cast the data to and from `unsigned char` instead.
+
+## `stepBased` Encoding of Iterations
+
+The `iterationEncoding` mode `stepBased` must be implemented via ADIOS steps.
+
+## Datasets
+
+An openPMD **data set** is represented by an group prefix that contains an ADIOS variable `__data__`.
```

Comment:
Generally, we will stick with a schema that is similar to the current schema (datasets and attributes are distinguished by using different backend features (variables and attributes)), but with two additions:

* Attributes can be variable now too
* We will use some protocol for identifying if a group is active in the current step

Since that updated schema is not yet implemented, I'd suggest we don't describe this just yet. I would not like to standardize something that in the end turns out to not work well.

Reply:
I've started exploring that new schema [here](https://github.com/openPMD/openPMD-api/pull/1310).